tests/Unit/Domain/Creators/Test_BinaryCompactObject.cpp

```diff
@@ -255,8 +262,38 @@ void test_connectivity() {
                {15, Direction<3>::lower_zeta()},
                {16, Direction<3>::lower_zeta()},
                {17, Direction<3>::lower_zeta()}}});
+      if (final_block.is_time_dependent()) {
+        expected_excision_spheres.at("ExcisionSphereB")
+            .inject_time_dependent_maps(
+                final_block.moving_mesh_grid_to_inertial_map().get_clone());
+      }
+    }
+    const auto& excision_spheres = domain.excision_spheres();
+    CHECK(excision_spheres == expected_excision_spheres);
+
+    const auto check_excision_sphere_map =
+        [&binary_compact_object](const ExcisionSphere<3>& excision_sphere) {
+          const double time = 0.0;
```

Comment:
I don't think this code runs at all because the `binary_compact_object` isn't time dependent here, right? Move to `test_bbh_time_dependent_factory` below?

Reply:
Done